tests/phpunit/integration/tests/Taxonomy/Taxonomy_Base.php

```diff
@@ -53,6 +57,9 @@ public function test_register_taxonomy(): void {
 	 */
 	public function test_unregister_taxonomy(): void {
 		$this->instance->register();
+		/**
+		 * @var string $slug
+		 */
 		$slug = $this->get_private_property( $this->instance, 'taxonomy_slug' );
```

Comment:
Instead of testing the private property, we can just use `$slug = $this->instance->get_taxonomy_slug()` to get the slug.

```suggestion
		$slug = $this->instance->get_taxonomy_slug();
```

Reply:
Removed in lots of instances of this.